content/docs/command-reference/run.md

```diff
@@ -11,7 +11,7 @@ usage: dvc run [-h] [-q | -v] [-d <path>] [-o <path>] [-O <path>]
                [-w <path>] [--no-exec] [--overwrite-dvcfile]
                [--no-run-cache] [--no-commit]
                [--outs-persist <path>] [--outs-persist-no-cache <path>]
-               [--always-changed]
+               [--always-changed] [--external]
```

Comment:
Idk if it makes sense to update this based on the 0.94 usage. The merged version will correspond to nothing 😋 e.g. the .dvc file name option is now `-n`, not `-f` as in this version. Same for the actual `dvc run` examples which probably don't work in 1.0 anyway.

Better to wait until after the 1.0 updates of add and especially run before trying to merge this one.

Reply:
UPDATE: This is being done in #1411 (which BTW steals the dvc add changes from this PR, sorry).

Reply:
I meant in #1420 actually. 1411 is for dvc add. Both steal Ruslan's work a little bit 😢

Reply:
This can be continued after 1411 and 1420 I think. The problem we have rn is multiple overlapping PRs associated to 1.0 updates 😢 

Other than add and run being outdated in this branch, is this still WIP, BTW? It not, I can take it over so you don't have to keep track of this one. Thanks

Reply:
@jorgeorpinel You've marked it as WIP, so I suppose you wanted to add something. If nothing left - let's merge or close, whatever suits you.

Reply:
No, it was just to remember to wait for those other PRs, one of which is merged so this is pretty much mergeable now. Anyway don't worry about it I'll solve any conflicts if needed and merge it ASAP. It's not about me though